container/src/mounts.rs

```diff
@@ -42,6 +42,7 @@ impl Mounts {
             path.push(&mount.source);
 
             if let Some(code) = Command::new("umount")
+                .arg("-l")
```

Comment:
I added this argument prevent the cleanup from errors due to busy mounts.

Reply:
It should not fail. Adding `-l` will hide an issue that needs to be fixed.
I'm not saying you should fix said issue in that PR, but you should not hide it so that it can be fixed later on.